mmv1/templates/terraform/examples/database_migration_service_connection_profile_alloydb.tf.erb

```diff
@@ -0,0 +1,58 @@
+data "google_project" "project" {
+}
+
+resource "google_compute_network" "default" {
+  name = "tf-test-alloydb-cp%{random_suffix}"
+}
+
+resource "google_compute_global_address" "private_ip_alloc" {
+  name          =  "private-ip-alloc%{random_suffix}"
+  address_type  = "INTERNAL"
+  purpose       = "VPC_PEERING"
+  prefix_length = 16
+  network       = google_compute_network.default.id
+  depends_on = [google_compute_network.default]
+}
+
+resource "google_service_networking_connection" "vpc_connection" {
+  network                 = google_compute_network.default.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.private_ip_alloc.name]
+  depends_on = [google_compute_global_address.private_ip_alloc]
+}
+
+
+resource "google_database_migration_service_connection_profile" "<%= ctx[:primary_resource_id] %>" {
+	location = "us-central1"		
+	connection_profile_id = "<%= ctx[:vars]["profile"] %>"
+	display_name = "<%= ctx[:vars]["profile"] %>_display"
+	labels	= { 
+					testanno = "testanno" 
+				}
+	alloydb {
+		cluster_id = "dbmsalloycluster%{random_suffix}"
+		settings {
+			initial_user {
+				user = "alloyuser%{random_suffix}"
+				password = "alloypass%{random_suffix}"
+			}
+			vpc_network = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
+			labels	= { 
+					testalloylabela = "testalloylabelb" 
+				}
+			primary_instance_settings {
+				id = "priminstid"
+				machine_config {
+					cpu_count = 2
+				}
+				database_flags = { 
+				}
+				labels	= { 
+					testalloyinstlabela = "testalloyinstlabelb" 
+				}
+			}
+		}
+	}
```

Comment:
```suggestion
  location = "us-central1"		
  connection_profile_id = "<%= ctx[:vars]["profile"] %>"
  display_name = "<%= ctx[:vars]["profile"] %>_display"
  labels = { 
    testanno = "testanno" 
  }
  alloydb {
    cluster_id = "dbmsalloycluster%{random_suffix}"
    settings {
      initial_user {
        user = "alloyuser%{random_suffix}"
        password = "alloypass%{random_suffix}"
      }
      vpc_network = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
      labels  = { 
          testalloylabela = "testalloylabelb" 
        }
      primary_instance_settings {
        id = "priminstid"
        machine_config {
          cpu_count = 2
        }
        database_flags = { 
        }
        labels  = { 
          testalloyinstlabela = "testalloyinstlabelb" 
        }
      }
    }
  }
```
don't wanna mix tabs and spaces in an example, folks tend to copy and paste these

Reply:
updated all to spaces as suggested and removed unneeded indentations